akka-remote/src/main/scala/akka/remote/transport/netty/NettySSLSupport.scala

```diff
@@ -119,12 +128,16 @@ private[akka] object NettySSLSupport {
    * Construct a SSLHandler which can be inserted into a Netty server/client pipeline
    */
   def apply(settings: SSLSettings, log: MarkerLoggingAdapter, isClient: Boolean): SslHandler = {
-    val sslEngine = settings.getOrCreateContext(log).createSSLEngine // TODO: pass host information to enable host verification
```

Comment:
The comment got lost. I think it might still make sense to keep it here? Or is it in preparation to the other PR?